index.js

```diff
@@ -39,16 +39,19 @@ var totalMetaCount = 0;
 
 function iterator (i, end, payload) {
   if (i < end) {
-    var url = s3.getPublicUrlHttp(params.s3Params.Bucket, payload[i].Key);
-    generateMeta(url, config.platform, config.provider, config.contact, function (err, msg) {
-      if (err) {
-        console.log(err);
-        return;
-      }
-      console.log(msg);
-      totalMetaCount++;
-      iterator(i + 1, end, payload);
-    });
+    var f = payload[i].Key.split('.');
+    if (f[f.length - 1].toUpperCase() === 'TIF'){
```

Comment:
Should the `.tiff` extension be supported as well? Just ran into a similar issue in GeoTrellis where a user had a `.tiff` and it wasn't getting picked up by an ingest process. It might make sense to explicitly require `.tif` for OIN for simplicity, though.